src/components/SecondaryPanes/XHRBreakpoints.js

```diff
@@ -179,23 +179,25 @@ class XHRBreakpoints extends Component<Props, State> {
   };
 
   renderBreakpoints = () => {
-    const { showInput, xhrBreakpoints } = this.props;
+    const { xhrBreakpoints } = this.props;
+    const explicitXhrBreakpoints = xhrBreakpoints.filter(bp => bp.path !== "");
+
     return (
       <ul className="pane expressions-list">
-        {xhrBreakpoints.map(this.renderBreakpoint)}
-        {(showInput || !xhrBreakpoints.size) &&
+        {explicitXhrBreakpoints.map(this.renderBreakpoint)}
+        {explicitXhrBreakpoints.size === 0 &&
```

Comment:
we might want to keep `showInput` for the case when we have few bps and want to add more